FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -57,11 +59,10 @@ namespace edm {
             bool m_debug;
             std::atomic_flag m_shouldUpdate;
             time_t m_beginJob = 0;
-            time_t m_updateInterval = m_defaultUpdateInterval;
+            time_t m_updateInterval;
             float m_emaInterval = m_defaultEmaInterval;
             float m_rate = 0;
             static constexpr float m_defaultEmaInterval = 15*60; // Time in seconds to average EMA over for event rate.
-            static constexpr unsigned int m_defaultUpdateInterval = 3*60;
```

Comment:
I think you still need to have a line
```
time_t CondorStatusService::m_defaultUpdateInterval;
```
So that the compiler will actually create a memory location for the value.